src/plugins/wb-onpage-submit/wb-onpage-submit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @title WET-BOEW wb-onpage-submit
+ * @overview This plugin implements AJAX request for form data
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee, @namjohn920
+ **/
+
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-onpage-submit",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		elm.addEventListener( "submit", function( e ) {
+			e.preventDefault();
+			var $elm = $( this );
+			try {
+				$.ajax( {
+					type: $elm.attr( "method" ),
+					url: $elm.attr( "action" ),
+					data: $elm.serialize() // serializes the form's elements.
```

Comment:
we will need to manage the serialize separatell because the button value used to submit is out of scope of the serialization because jQuery said the button are out of scope which is true unless that button is used to submit. So we need to detect if "elm" was the button used to do the submission and if so, we need to serialize it's value.